openedx/core/djangoapps/discussions/models.py

```diff
@@ -21,18 +21,114 @@
 
 log = logging.getLogger(__name__)
 
-
 DEFAULT_PROVIDER_TYPE = 'legacy'
+FEATURES = {
+    'ADVANCED_IN_CONTEXT_DISCUSSION': 'advanced_in_context_discussion',
+    'ANONYMOUS_POSTING': 'anonymous_posting',
+    'AUTOMATIC_LEARNER_ENROLLMENT': 'automatic_learner_enrollment',
+    'BLACKOUT_DISCUSSION_DATES': 'blackout_discussion_dates',
+    'COMMUNITY_TA_SUPPORT': 'community_ta_support',
+    'COURSE_COHORT_SUPPORT': 'course_cohort_support',
+    'DIRECT_MESSAGES_FROM_INSTRUCTORS': 'direct_messages_from_instructors',
+    'DISCUSSION-PAGE': 'discussion-page',
+    'DISCUSSION_CONTENT_PROMPTS': 'discussion_content_prompts',
+    'EMAIL_NOTIFICATIONS': 'email_notifications',
+    'EMBEDDED-COURSE-SECTIONS': 'embedded-course-sections',
+    'GRADED_DISCUSSIONS': 'graded_discussions',
+    'IN_PLATFORM_NOTIFICATIONS': 'in_platform_notifications',
+    'INTERNATIONALIZATION_SUPPORT': 'internationalization_support',
+    'LTI': 'lti',
+    'LTI_ADVANCED_SHARING_MODE': 'lti_advanced_sharing_mode',
+    'LTI_BASIC_CONFIGURATION': 'lti_basic_configuration',
+    'PRIMARY_DISCUSSION_APP_EXPERIENCE': 'primary_discussion_app_experience',
+    'QUESTION_&_DISCUSSION_SUPPORT': 'question_&_discussion_support',
+    'REPORT/FLAG_CONTENT_TO_MODERATORS': 'report/flag_content_to_moderators',
+    'RESEARCH_DATA_EVENTS': 'research_data_events',
+    'SIMPLIFIED_IN_CONTEXT_DISCUSSION': 'simplified_in_context_discussion',
+    'USER_MENTIONS': 'user_mentions',
+    'WCAG-2.1': 'wcag-2.1',
+    'WCAG_2.0_SUPPORT': 'wcag_2.0_support',
+}
+
 PROVIDER_FEATURE_MAP = {
     'legacy': [
-        'discussion-page',
-        'embedded-course-sections',
-        'wcag-2.1',
+        FEATURES['DISCUSSION-PAGE'],
+        FEATURES['WCAG-2.1'],
+        FEATURES['AUTOMATIC_LEARNER_ENROLLMENT'],
+        FEATURES['WCAG_2.0_SUPPORT'],
+        FEATURES['INTERNATIONALIZATION_SUPPORT'],
+        FEATURES['ANONYMOUS_POSTING'],
+        FEATURES['REPORT/FLAG_CONTENT_TO_MODERATORS'],
```

Comment:
No need to translate everything as a key you could simply have this key as `REPORT_FLAG_CONTENT_TO_MODERATORS` and avoid `/` in the keyname.